src/component/Atag.js

```diff
@@ -0,0 +1,24 @@
+import AtagImg from "./AtagImg";
+function Atag({
+  href = "",
+  target = "",
+  rel = "",
+  name = "",
+  className = "",
+  contents = "",
+}) {
+  return (
+    <>
+      {name === "" || null ? (
```

Comment:
name이 [falsy 한 값](https://developer.mozilla.org/ko/docs/Glossary/Falsy)인지로 판단하면 충분할 거같네요. 

```suggestion
      {!name ? (
```